sdks/python/scripts/generate_pydoc.sh

```diff
@@ -91,6 +99,9 @@ import sys
 
 import sphinx_rtd_theme
 
+import numpy
+import IPython
```

Comment:
Why are these imports required?

Reply:
This is workaround for double-import in Sphinx-autodoc.
Without `import numpy` here, RuntimeError like https://issues.apache.org/jira/browse/BEAM-7847 will be occurred.
And I deleted `import IPython` because I realized that it is not necessary.